Week3/akif-week3.js

```diff
@@ -0,0 +1,158 @@
+favoriteAnimals.splice(3, 1);
+console.log(favoriteAnimals);
+console.log('The item you are looking for is at index: ' + favoriteAnimals.indexOf('meerkat'));
+favoriteAnimals.splice(favoriteAnimals.indexOf('meerkat'), 1);
+
+//  MORE JAVASCRIPT ! ------------------------------------------------
+// -----1111111111111111111-----------
+function getSum(a, b, c) {
+  return a + b + c;
+}
+
+// -----222222222222222222-------------
+function colorCar(color) {
+  return 'a ' + color + ' car';
+}
+
+//-----3333333333333333333--------------
+const myBooks = {
+  Dostoyevsky: 'Crime and Punishment',
+  'Franz Kafka': 'The Trial',
+  'Anne Frank': 'Diarys',
+};
+
+function getBook(library) {
+  for (let book in library) {
+    console.log(book + ' - ' + library[book]);
+  }
+}
+getBook(myBooks);
+
+//  -----444444444444444444------------------
+const vehicleCodes = {
+  1: 'car',
+  2: 'motorbike',
+};
+
+function vehicleType(color, code) {
+  console.log('A ' + color + ' ' + vehicleCodes[code]);
+}
+
+vehicleType('blue', 2);
+
+//  _-------555555555555555555--------------------
+console.log(3 === 3 ? 'yes' : 'no');
+
+//  -------66666666666666666666---------------------
+function vehicle(color, code, age) {
+  if (age > 0) {
+    age = 'used';
+  } else {
+    age = 'new';
+  }
+  console.log('A ' + color + ' ' + age + ' ' + vehicleCodes[code]);
+}
+
+vehicle('blue', 1, 5);
+
+// -------777777777777  &&  888888888888888--------------------
+let vehicleList = ['car', 'motorbike', 'caravan', 'bike', 'truck'];
+
+console.log('Third element of the array is ' + vehicleList[2]);
+
+//  --------9999999999999999999999 -----------------------------
+function vehicle1(color, code, age) {
+  if (age > 1) {
+    age = 'used';
+  } else {
+    age = 'new';
+  }
+  console.log('A ' + color + ' ' + age + ' ' + vehicleList[code]);
+}
+
+vehicle1('green', 3, 1);
+
+//-------10 10 10 10 10 10 10 10 ---------------------
+
+// function advertisement(arrname) {
+//   for (let i = 0; i < arrname.length; i++) {
+//     arrname[i] += 's';
+//   }
+
+//   for (let i = 0; i < arrname.length - 2; i++) {
+//     arrname[i] += ',';
+//   }
+
+//   arrname[arrname.length - 2] += ' and';
+
+//   console.log("Amazing Joe's Garage, we service " + arrname.join(' ') + '.');
+// }
+
+function advertisement(arrname) {
+  let vehicles1 = arrname.slice(0, arrname.length - 1).join('s, ');
+  let lastVehicle = arrname[arrname.length - 1];
+
+  console.log(`"Amazing Joe's Garage, we service ${vehicles1}s and ${lastVehicle}s."`);
+}
+
+advertisement(vehicleList);
+
+// --------11 11 11 111 11 11 11 11 ----------------------
+vehicleList.push('ship');
+advertisement(vehicleList);
+
+// ---12 12 12 12 12 12 12 12 // 13 13 13 // 14 14 14 14 -------------------------
+const teachers = {};
+teachers.HTML = 'Philipp Beau';
+teachers.CSS = 'Rob van Kruijsdijk';
+teachers.GIT = 'Unmesh Joshi';
+teachers.JS = 'Yash Kapila';
+console.log(teachers);
```

Comment:
Hi Akif, is there a reason why you didn't put all the values during the object declaration itself i.e
```
const teachers = {
  HTML: 'Philipp Beau',
  CSS: 'Rob van Kruijsdijk',
  GIT: 'Unmesh Joshi',
  JS: 'Yash Kapila'
};
```

Reply:
i know to do this but i think the directives in the homework was not clear. 
"12.Create an empty object.
13.Create an object that contains the teachers that you have had so far for the different modules.
14.Add a property to the object you just created that contains the languages that they have taught you." 😃